ipn/ipnlocal/local.go

```diff
@@ -2232,6 +2232,49 @@ func (b *LocalBackend) authReconfig() {
 	b.initPeerAPIListener()
 }
 
+// shouldUseOneCGNATRoute returns whether we should prefer to make one big
```

Comment:
"reports whether" (sorry)

I can still hear Rob Pike saying "The function returns regardless."